### ISSUE TITLE
Wrong Tag

### ISSUE THREAD

**Issue by mms-gianni** [anbraten/gitea-js#4]
Hi

I am using this library in [Kubero](https://github.com/kubero-dev/kubero).  As I have seen, you have a typo in Version v1.17.0 and version 1.16.x - 1.19- seem to be the same. 

https://github.com/anbraten/gitea-js/releases/tag/v.17.0

**Comment by anbraten** [comment]
@mms-gianni Couldn't find the issue directly. The schemas of the different Gitea versions are stored in `src/schemas/<version>.json` and they seem to have the correct version inside as far as I can tell. What do you mean by typo in version?

The build logs of for example `0.17` tag seem to be fine from a first look as well:

```
> gitea-js@0.0.0 generate /home/runner/work/gitea-js/gitea-js
> tsx contrib/generate.ts

Generating gitea-js version "1.17.1" for Gitea "1.17" ...
✨   swagger-typescript-api(12.0.2),pnpm/7.16.1 npm/? node/v1[8](https://github.com/anbraten/gitea-js/actions/runs/3504737945/jobs/5870609236#step:7:9).[12](https://github.com/anbraten/gitea-js/actions/runs/3504737945/jobs/5870609236#step:7:13).0 linux x64
✨   try to read templates from directory "/home/runner/work/gitea-js/templates/default"
⚠️    "api" template not found in "/home/runner/work/gitea-js/templates/default" 
⚠️    Code generator will use the default template
⚠️    "route types" template not found in "/home/runner/work/gitea-js/templates/default" 
⚠️    Code generator will use the default template
✨   try to get swagger by path "/home/runner/work/gitea-js/gitea-js/src/schemas/v1.[17](https://github.com/anbraten/gitea-js/actions/runs/3504737945/jobs/5870609236#step:7:18).json"
☄️    start generating your typescript api
✅   typescript api file api.ts created in /home/runner/work/gitea-js/gitea-js/src
Generating docs for ...
Documentation generated at ./docs

Setting package.json version to "1.17.1" ...

Done.
```

**Comment by mms-gianni** [comment]
Sorry for the to short description. I was a bit in a hurry. But this image explains it best: 

I am pretty sure there is a "1" missing. And the revision numbers are the same for several tags. 

![image](https://user-images.githubusercontent.com/2052196/204843346-ce48edf4-dc12-4a7e-9037-70283f201f6a.png)

**Comment by anbraten** [comment]
They all share the same commit as I recently introduced support for different Gitea versions. Basically all supported versions and their corresponding schemas are stored in the `src/schemas` folder. I did a commit for all latest schemas of the different versions of Gitea and pushed those, so they all are based on the same commit, but the tag name sets the version the Github actions workflow is releasing. Did you tried using the lib and had a problem with a specific version?
